lib/puck/dependency_resolver.rb

```diff
@@ -32,11 +33,11 @@ def resolve_gem_dependencies(options = {})
 
     private
 
-    def contained_bundler(gem_home, gemfile, lockfile, groups)
+    def contained_bundler(gem_home, gemfile, lockfile, groups, app_dir)
       Bundler.with_clean_env do
         scripting_container = Java::OrgJrubyEmbed::ScriptingContainer.new(Java::OrgJrubyEmbed::LocalContextScope::SINGLETHREAD)
         scripting_container.compat_version = Java::OrgJruby::CompatVersion::RUBY1_9
-        scripting_container.current_directory = Dir.pwd
```

Comment:
I'm hearing you talking about this as I write, but I'll go ahead and make this comment anyway.

Why not use `File.dirname(gemfile)` and avoid the extra variable?